src/UploadedFile.php

```diff
@@ -212,34 +212,17 @@ public function getClientMediaType(): ?string
      *
      * @param StreamInterface $source Stream to read from
      * @param StreamInterface $dest Stream to write to
-     * @param int $maxLen Maximum number of bytes to read. Pass -1
-     *                    to read the entire stream
      *
      * @throws \RuntimeException on error
      */
-    private function copyToStream(StreamInterface $source, StreamInterface $dest, $maxLen = -1): void
+    private function copyToStream(StreamInterface $source, StreamInterface $dest): void
```

Comment:
We are never using the third parameter, That is why I could remove the dead code